From: Ben Hutchings <ben@decadent.org.uk>
Subject: ast: Disable autoload
Date: Mon, 24 Dec 2012 00:05:07 +0000

This kernel driver is incompatible with older versions of
xserver-xorg-video-ast, so do not autoload it yet.  It will
have to be loaded either by local configuration or as part of a
package of a compatible X driver.

--- a/drivers/gpu/drm/ast/ast_drv.c
+++ b/drivers/gpu/drm/ast/ast_drv.c
@@ -58,7 +58,7 @@ static DEFINE_PCI_DEVICE_TABLE(pciidlist
 	{0, 0, 0},
 };
 
-MODULE_DEVICE_TABLE(pci, pciidlist);
+/*MODULE_DEVICE_TABLE(pci, pciidlist);*/
 
 static int __devinit
 ast_pci_probe(struct pci_dev *pdev, const struct pci_device_id *ent)
